client/src/components/edit-formula-dialog/SetFormulaParams.tsx

```diff
@@ -3,12 +3,15 @@
 // SPDX-License-Identifier: MIT
 
 import { AttainmentData, FormulaData } from 'aalto-grades-common/types';
+import { TextField } from '@mui/material';
 import PropTypes from 'prop-types';
+import { ChangeEvent } from 'react';
 
 import SubAttainment from './SubAttainment';
-
 import StyledBox from '../edit-formula-dialog/StyledBox';
 
+import { getParamLabel } from '../../utils';
+
 export default function SetFormulaParams(props: {
```

Comment:
The editing form could show the currently set params if the new formula is same as current one